code/ai/ai_profiles.cpp

```diff
@@ -472,6 +472,18 @@ void parse_ai_profiles_tbl(const char *filename)
 
 				set_flag(profile, "$use actual primary range:", AI::Profile_Flags::Use_actual_primary_range);
 
+				if (optional_string("$constant radii for subsystem path points:")) {
+					int path_radii;
+					stuff_int(&path_radii);
+					if (path_radii) {
+						profile->subsystem_path_radii = path_radii;
+					} else {
+						mprintf(("Warning: \"$constant radii for subsystem path points:\" should be greater than 0 (read %i). Value will not be used. ", path_radii));
+					}
```

Comment:
With this if() block, any nonzero radius, even if negative, will be used.  Did you mean to check `if (path_radii > 0)`?

Reply:
Correct, it should be > 0. Thanks!